cost-model/src/cost_model.rs

```diff
@@ -624,45 +616,6 @@ mod tests {
         assert_eq!(1, data_bytes_cost);
     }
 
-    #[test]
-    fn test_cost_model_with_failed_compute_budget_transaction() {
```

Comment:
I think we can keep this test but change it so that there is a compute budget ix that fails in `sanitize_and_convert_to_compute_budget_limits` rather than in `ComputeBudgetInstructionDetails::try_from` since any errors in `try_from` would be caught earlier when `RuntimeTransaction` is created. Also, shouldn't `get_transaction_cost` be updated to use a runtime tx to avoid reprocessing compute budget ixs?

Reply:
Added this test back as you described - ended up just updating cost-model code to use `RuntimeTransaction` as well to make it so I don't need to update the test again next PR